server/core/config.py

```diff
@@ -3,11 +3,14 @@
 from pydantic import BaseSettings
 
 API_V1_STR = "/api/v1"
+COASTLINES_V1_STR = API_V1_STR + "coastlines"
```

Comment:
We'd need a `/` before "coastlines" to prevent an accidental bug once we start using these values.

If you were to print `COASTLINES_V1_STR`, the output would be `/api/v1coastlines`.

**Style Note**: instead of concatenating strings with a plus, use Python [f-strings](https://realpython.com/python-f-strings/) instead.

Reply:
Modified using fstring formating